### PR TITLE
Documentation: Change url_prefix to match actual endpoint Fix #3630

### LINKED ISSUE #3630: REST API documentation does not reflect some actual endpoints
Motivation
----------
Some REST endpoints path in the documentation differs from the actual API path. Several instances of this:

1. `/rules` endpoint, listed in the docs as `/rule`
2. `/config` endpoint, listed in the docs as `/configs`
3. `/tmp_dids` endpoint, listed in the docs as `/temporary_dids`
4. `/accountlimits` endpoint, listed in the docs as `/accounts`

Additionally, there are endpoints that exist in the source but lack the accompanying docs:
1. `/credentials` endpoint
2. `/export` endpoint
3. `/import` endpoint

Modification
------------

Several files need to be edited/added:
- `lib/rucio/web/rest/flaskapi/v1/{account_limit|config|rule|temporary_dids}.py` need to be edited to change the `doc_app.register_blueprint(bp, url_prefix='/<endpoint prefix>')` portion.
- Add new `.rst` files

### PR DESCRIPTION
**Pull request by didithilmy**
Documentation: Change url_prefix to match actual endpoint Fix #3630

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -43,6 +43,7 @@ Individual contributors to the source code
 - Andrew Lister <andrew.lister@stfc.ac.uk>, 2019
 - Aristeidis Fkiaras <aristeidis.fkiaras@cern.ch>, 2019
 - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020
+- Muhammad Aditya Hilmy <didithilmy@gmail.com>
```
Comment (bari12) [inline AUTHORS.rst]: Add year

**Review by bari12 — CHANGES_REQUESTED** [review]
Also tests are failing; But that could be related to #3640

**Comment by bziemons** [comment]
> Also tests are failing; But that could be related to #3640

Looking at the error of the syntax test, it is unrelated to #3640

**Comment by didithilmy** [comment]
> Also tests are failing; But that could be related to #3640

I'm not sure why it happened. If I see the Travis logs, the files that are causing the error weren't even included in my changes. As I recall, when I checked out the repo from `master` it was already failed. Any suggestion on how to fix this?

**Comment by bziemons** [comment]
The offending line seems to be this one: https://github.com/didithilmy/rucio/blob/9cd521385549a312442cb8d88106ed0f6852fd1d/lib/rucio/web/rest/flaskapi/v1/exporter.py#L72

which is the same in `master` and `next`, but this error is probably coming up now because of this change: https://github.com/rucio/rucio/pull/3631/files#diff-72e4556e2f5d8168cb86e424174fe576

**Comment by bziemons** [comment]
> Any suggestion on how to fix this?

Can you try to change the `bp.add_url_rule('', …` to `bp.add_url_rule('/', …` in exporter.py and importer.py in lib/rucio/web/rest/flaskapi/v1 and see if that fixes the issue?

**Comment by didithilmy** [comment]
> The offending line seems to be this one: https://github.com/didithilmy/rucio/blob/9cd521385549a312442cb8d88106ed0f6852fd1d/lib/rucio/web/rest/flaskapi/v1/exporter.py#L72
> 
> which is the same in `master` and `next`, but this error is probably coming up now because of this change: https://github.com/rucio/rucio/pull/3631/files#diff-72e4556e2f5d8168cb86e424174fe576

Okay that makes sense. I've pushed the changes as suggested by @bziemons

**Review by bari12 — APPROVED** [review]

**Pull request merged**